cmake/linux/debian/lmms-vst-full.install.in

```diff
@@ -0,0 +1 @@
+usr/lib/lmms/RemoteVstPlugin*
```

Comment:
It doesn't hurt to keep this as an `.in` file, but just an FYI, if you aren't providing it parameters, it technically doesn't have to be configured.  Feel free to leave this as-is if you feel this may take some CMake params down the road.